spec/jenkins_pipeline/app_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'spec_helper'
+
+describe JenkinsPipeline::App do
+
+  it "Returns urls from all pipeline config files" do
+    get '/api/pipelines/ids'
+    # expect(subject).to have_received(:configuration)
```

Comment:
That's fine, let's keep it. I would just delete the comment and rename the test. What do you think?